deps/wazuh_testing/wazuh_testing/modules/aws/data_generator.py

```diff
@@ -94,22 +103,150 @@ def get_data_sample(self) -> dict:
                     'sharedEventID': str(uuid4())
                 }
             ]
-        }
+        })
+
+
+class VPCDataGenerator(DataGenerator):
+    BASE_PATH = f'{cons.AWS_LOGS}/{cons.RANDOM_ACCOUNT_ID}/{cons.VPC_FLOW_LOGS}/{cons.US_EAST_1_REGION}/'
+    BASE_FILE_NAME = f'{cons.RANDOM_ACCOUNT_ID}_{cons.VPC_FLOW_LOGS}_{cons.US_EAST_1_REGION}_'
+
+    def get_filename(self, prefix=None, **kwargs) -> str:
+        """Return the filename in the cloudtrail format
+        <prefix>/AWSLogs/<suffix>/<organization_id>/<account_id>/vpcflowlogs/<region>/<year>/<month>/<day>
+        """
+        now = datetime.now()
+        path = f'{self.BASE_PATH}{now.strftime(cons.PATH_DATE_FORMAT)}/'
+        name = (
+            f'{self.BASE_FILE_NAME}{cons.FLOW_LOG_ID}_{now.strftime(cons.FILENAME_DATE_FORMAT)}_{abs(hash(now))}'
+            f'{cons.LOG_EXT}'
+        )
+
+        return f'{path}{name}'
+
+    def get_data_sample(self) -> str:
```

Comment:
Docstring is missing

Reply:
Please disregard this, already has been fixed in https://github.com/wazuh/wazuh-qa/pull/3699/.